packages/core/test/unit/html/NodeProcessor.data.ts

```diff
@@ -38,29 +38,6 @@ export const PROCESS_PANEL_HEADER_SLOT_TAKES_PRIORITY_EXPECTED = `
 
 export const PROCESS_PANEL_HEADER_SLOT_TAKES_PRIORITY_WARN_MSG = "panel has a header slot, 'header' attribute has no effect.";
 
-export const PROCESS_PANEL_HEADER_NO_OVERRIDE = `
-<panel header="# Lorem ipsum" alt="**strong alt**">
-  <div slot="header">
-    This existing header slot should be preserved in favour over header attribute.
-  </div>
-  Header attribute should not be inserted under panel since there is both an alt attribute and header slot,
-  but should be deleted.
-  Alt attribute should be inserted under panel as slot.
-</panel>
-`;
-
-export const PROCESS_PANEL_HEADER_NO_OVERRIDE_EXPECTED = `
-<panel><template #_alt><p><strong>strong alt</strong></p>
-</template>
-  <template #header><div>
-    This existing header slot should be preserved in favour over header attribute.
-  </div></template>
-  Header attribute should not be inserted under panel since there is both an alt attribute and header slot,
-  but should be deleted.
-  Alt attribute should be inserted under panel as slot.
-</panel>
-`;
-
 // Post Process
 
 export const POST_PROCESS_PANEL_ID_ASSIGNED_USING_HEADER_SLOT = `
```

Comment:
Let me know if I'm missing something here, but it doesn't look like this one has an alt slot for it to test for it being inserted under panel as stated in your earlier comment?

Reply:
:O You are right - I think I got confused by the text in the constant. Will delete the other constant then